pylint_odoo/checkers/modules_odoo.py

```diff
@@ -838,4 +844,22 @@ def _check_xml_attribute_translatable(self):
                     ("%s:%d" % (xml_file, record.sourceline), 'xml_id'))
         if self.msg_args:
             return False
+
+    def _check_useless_attribute_non_translatable(self):
+        """The xml attribute translation="off" is not necessary when version is
+            different to '8.0'
+            Example  <attribute name="groups"
+                                translation="off">sale.group</attribute>
+        """
+        if (self.linter._all_options['valid_odoo_versions'].config
+                .valid_odoo_versions != ['8.0']):
```

Comment:
here could receive many versions because is a list.
`['8.0', '9.0']`
This check should be emitted just in case of that the value is exactly and item and is different to 8.0

I mean,
`... and len(self.linter._all_options['valid_odoo_versions'].config.valid_odoo_versions) == 1`